docs/source/serving/service-yaml-spec.rst

```diff
@@ -54,23 +54,18 @@ Available fields:
         # replica. If not specified, SkyServe will **NOT** scale down your service.
         qps_lower_threshold: 2
 
-      # Also, for convenience, we have a simplified version of replica policy that
-      # use fixed number of replicas. Just use the following syntax:
+      # Simplified version of replica policy that uses a fixed number of
+      # replicas:
       replicas: 2
 
-      # Controller resources (optional). This describe the resources to use for
-      # the controller. Default to a 4+ vCPU instance with 100GB disk.
-      controller_resources:
-        cloud: aws
-        region: us-east-1
-        instance_type: p3.2xlarge
-        disk_size: 256
+    # Besides the `service` section, the rest is a regular SkyPilot task YAML,
+    # which describes each replica:
 
     resources:
-      # Port to run your service (required). This port will be automatically exposed
-      # by SkyServe. You can access your service at http://<endpoint-ip>:<port>.
+      # Port to run your service on each replica (required). This port will be
+      # automatically exposed by SkyServe.
```

Comment:
```suggestion
      # automatically exposed to the public internet by SkyServe.
```
not sure if we want to include it; could discuss

Reply:
Done.